test/dom/element.test.js

```diff
@@ -208,6 +208,87 @@ describe('Document', () => {
 		expect(doc.childNodes.toString()).toBe(`<A/><B/><C/>`)
 	})
 
+	it('ElementTraversal', () => {
```

Comment:
I didn't dig into every line to understand it, but I would prefer to have multiple `it`s that describe "independent" behaviour/scenarios and if some things only make sense to be tested together, of course leave them together. But my guess is that your comments might indicate some "sections"?

I don't have an issue with nested `describe` blocks, so maybe it's like 
```suggestion
	describe('element traversal', () => {
```
and it contains the different scenarios as `it`s?